### DISCUSSION TITLE
Docs for Paxcounter project

### DISCUSSION THREAD

**Discussion by t-huyeng** [cyberman54/ESP32-Paxcounter#918] [Show and tell]
I am working on some docs or lets say more on teleporting the current readme into a nice looking webpage. 

See working branch: https://github.com/t-huyeng/ESP32-Paxcounter/tree/mkdocs
See current Webpage: https://t-huyeng.github.io/ESP32-Paxcounter/

I am using [Mkdocs Material](https://squidfunk.github.io/mkdocs-material/) and a [GitHub Workflow ](https://github.com/t-huyeng/ESP32-Paxcounter/blob/mkdocs/.github/workflows/docs.yml) to publish the files to a separate branch (`gh-pages`) which is then used for GitHub-Pages. 

Would love to here some thoughts about that! 

Todos
- [x] add/create logo
- [x] add examples for custom sensors etc. (see #917)
- [x] add more "real" code snippets (see [external files](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#embedding-external-files))  see: https://t-huyeng.github.io/ESP32-Paxcounter/getting-started/
- [x] favicon
- [x] Update Readme to link to docs and contain all needed Information. 
- [x] Link Readme to first Page of Docs

**Comment by t-huyeng** [comment]
First quick draft of a possible logo (included of the header in the docs): 
![image](https://user-images.githubusercontent.com/39559223/203020225-a9e57cc9-1fcd-4892-a4f6-44b307e4b8db.png)

**Reply by cyberman54** [reply]
We also need a startup screen with this logo. Form factor seems to have a good fit to TTGO etc. display. Should be easyly possible with the display libs, they use a common GFX command format. I will check which input format we need for the logo.

**Reply by cyberman54** [reply]
I am going to change the QR code displaying the DEV EUI with the logo. Because scanning the QR from the multiplexed display doesn't work well. So let's display the logo instead.

**Reply by cyberman54** [reply]
Maybe we need a logo with bold structures for this, to get this converted to pixel matrix.

**Reply by t-huyeng** [reply]
Good point with the display! Here is the SVG :
![paxcounter_logo_dark](https://user-images.githubusercontent.com/39559223/203053430-7d8f3d64-a61f-4af0-91cc-6a425099be85.svg)

More bold will probably be mean the `/` will not be really visible, but I will check what I can do with my limited inkscape skills ;)

**Reply by cyberman54** [reply]
We need .BMP, either for monochrome, as well for color displays.

**Reply by t-huyeng** [reply]
See : https://github.com/t-huyeng/ESP32-Paxcounter/tree/mkdocs/docs/assets for bmp files.

**Comment by t-huyeng** [comment]
Are you generally up to mkdocs or is this not wanted for this project?

**Reply by cyberman54** [reply]
I don't know anything on it yet. Don't see a problem, if there is no licence issue.

**Reply by t-huyeng** [reply]
Licence should not be a problem (MIT) and build for opensource projects like this one here.

**Comment by cyberman54** [comment]
> I am working on some docs or lets say more on teleporting the current readme into a nice looking webpage.

That would be a very useful add on, since documentation for this project has grown step-by-step, and thus is hard to read for new users. Thank you for putting effort on this!

**Reply by t-huyeng** [reply]
Perfect! 
I updated a few parts of the docs and added more code snippets with using the "real" files. 
I will try to resolve a few Todes soon, maybe update some images and then create a PR. If you have any suggestions let me know!

See: https://t-huyeng.github.io/ESP32-Paxcounter/ for the current status.

**Comment by t-huyeng** [comment]
see #919 for further updates.
